testcli.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	appsv1 "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1"
+	projectsv1 "github.com/openshift/client-go/project/clientset/versioned/typed/project/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	//"k8s.io/client-go/kubernetes"
```

Comment:
the only things which should be commented out in your code are actual comments. Don't leave commented-out imports and pieces of code lying around